tutorials/multiple-devices/multiple-device-stitch-nn/main.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+
+from depthai_nodes.node import ParsingNeuralNetwork
+from depthai_nodes.node import TilesPatcher, Tiling
+from utils.stitch import Stitch
+from utils.arguments import initialize_argparser
+import contextlib
+import depthai as dai
+import pathlib
+
+
+_, args = initialize_argparser()
+
+IMG_SIZES = {
+    "2160p": (3840, 2160),
+    "1080p": (1920, 1080),
+    "720p": (1280, 720),
+    "480p": (640, 480),
+    "360p": (640, 360),
+}
+IMG_SHAPE = IMG_SIZES[args.input_size]
+
+
+def createPipeline(pipeline):
+    camRgb = pipeline.create(dai.node.Camera).build(dai.CameraBoardSocket.CAM_A)
+    output = camRgb.requestOutput(
+        IMG_SHAPE, dai.ImgFrame.Type.NV12, dai.ImgResizeMode.CROP, fps=args.fps_limit
+    )
+    return pipeline, output
+
+
+with contextlib.ExitStack() as stack:
+    visualizer = dai.RemoteConnection(httpPort=8082)
+    deviceInfos = dai.Device.getAllAvailableDevices()
+    print("=== Found devices: ", deviceInfos)
+    outputs = []
+    pipelines = []
+
+    for deviceInfo in deviceInfos:
+        pipeline = stack.enter_context(dai.Pipeline())
+        device = pipeline.getDefaultDevice()
+
+        print("===Connected to ", deviceInfo.getDeviceId())
+        mxId = device.getDeviceId()
+        cameras = device.getConnectedCameras()
+        usbSpeed = device.getUsbSpeed()
+        eepromData = device.readCalibration2().getEepromData()
+        print("   >>> Device ID:", mxId)
+        print("   >>> Num of cameras:", len(cameras))
+        if eepromData.boardName != "":
+            print("   >>> Board name:", eepromData.boardName)
+        if eepromData.productName != "":
+            print("   >>> Product name:", eepromData.productName)
+
+        pipeline, output = createPipeline(pipeline)
+        pipelines.append(pipeline)
+
+        outputs.append(output)
+
+    script_path = pathlib.Path(
+        __file__
+    ).parent.resolve()  # get absolute path of this script
+    platform = device.getPlatform()  # get platform
+    # Get model based on platform type from yaml file
+    model_description = dai.NNModelDescription.fromYamlFile(
+        f"{script_path}/depthai_models/yolov6_nano.{platform.name}.yaml"
+    )
```

Comment:
Is this script path needed? Afaik this should work like this since .fromYamlFile() looks in `depthai_models` dir by default? Not 100% sure though.
```suggestion
    platform = device.getPlatform()  # get platform
    # Get model based on platform type from yaml file
    model_description = dai.NNModelDescription.fromYamlFile(
        f"yolov6_nano.{platform.name}.yaml"
    )
```

Reply:
yes its needed if you want to be able to run this script from anywhere. It checks inside `depthai_models` but only relative to the run path. So for example: qr with tiling example which does not have a full, absolute path implemented, has a problem:
```
(venv) mk@mkw ~/Documents/Git/oak-examples (nn-stitching) $ python3 tutorials/qr-with-tiling/main.py 
To connect to the DepthAI visualizer, open http://localhost:8082 in your browser
In case of a different client, replace 'localhost' with the correct hostname
Creating pipeline...
Traceback (most recent call last):
  File "/home/mk/Documents/Git/oak-examples/tutorials/qr-with-tiling/main.py", line 20, in <module>
    model_description = dai.NNModelDescription.fromYamlFile(
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Model file not found: `depthai_models/qrdet_nano.RVC2.yaml` | If you meant to use a relative path, prefix with ./ (e.g. `./qrdet_nano.RVC2.yaml`) | Also, make sure the file exists. Read the documentation for more information.
(venv) mk@mkw ~/Documents/Git/oak-examples (nn-stitching) $ cd tutorials/qr-with-tiling/
(venv) mk@mkw ~/Documents/Git/oak-examples/tutorials/qr-with-tiling (nn-stitching) $ python3 main.py 
To connect to the DepthAI visualizer, open http://localhost:8082 in your browser
In case of a different client, replace 'localhost' with the correct hostname
Creating pipeline...
Pipeline created.
```